cinderblock/integrate.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+
+# Script to trigger integration tests from the template-service repo
+# Requirements:
+# pip install circleclient
+
+import argparse
+import os
+from circleclient import circleclient
+
+
+def main():
+    args = __parse_args()
+    client = circleclient.CircleClient(args.token)
+
+    # Trigger a parametized build with the integration project with
+    # env variables to identify the trigger project and the version
+    client.build.trigger(args.owner, 'integration', args.branch,
+                         CINDERBLOCK_REPO_OWNER=args.owner,
+                         CINDERBLOCK_PROJECT_NAME=args.reponame,
+                         CINDERBLOCK_SHA=args.commit,
+                         CINDERBLOCK_BUILD_NUMBER=args.build,
+                         CINDERBLOCK_TARGET_URL=args.build_url)
+    print('Triggered integration for {} at commit: {}'.format(
+        args.reponame, args.commit))
+
+
+def __validate_args(args, argparser):
```

Comment:
OK I thought I tried doing exactly that but it didn't work as I expected, will test again